### ISSUE TITLE
Platypus: Code cell's `textarea` missing label

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#475]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the Code cell's `textarea` is missing a label.

This might be tricky to solve as it seems to be an issue with **Thebe**'s implementation of **CodeMirror**, not setting the `screenReaderLabel` configuration option.

We should still try and see if something can be done from our end to solve this.

---

https://dequeuniversity.com/rules/axe/4.3/label?application=axeAPI

---

**User impact:** Critical
**Disabilities affected:** Blind, Low Vision, Deafblind, Mobility
**Requirement(s):** Section 508
**WCAG Success Criteria:** 1.3.1 Info and Relationships, 4.1.2 Name, Role, Value

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
